server/src/Broadcast.hpp

```
@@ -252,6 +252,13 @@ namespace broadcast {
         broadcastToRoom(networkManager, roomClients, packet);
       }
 
+      static void broadcastPositionUpdateToRoom(
+          network::ServerNetworkManager &networkManager,
+          const std::vector<std::shared_ptr<server::Client>> &roomClients,
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)